server/controllers/feeds.ts

```diff
@@ -207,43 +228,304 @@ async function generateVideoFeedForSubscriptions (req: express.Request, res: exp
   return sendFeed(feed, req, res)
 }
 
+async function generateVideoPodcastFeed (req: express.Request, res: express.Response) {
+  const start = 0
+  const account = res.locals.account
+  const videoChannel = res.locals.videoChannel
+  const nsfw = buildNSFWFilter()
+
+  const { name, description, imageUrl, accountImageUrl, email, link, accountLink } = await buildFeedMetadata({ videoChannel })
+
+  const options = {
+    accountId: account ? account.id : null,
+    videoChannelId: videoChannel.id
+  }
+
+  const server = await getServerActor()
+  const { data } = await VideoModel.listForApi({
+    start,
+    count: CONFIG.FEEDS.VIDEOS.COUNT,
+    sort: '-publishedAt',
+    displayOnlyForFollower: {
+      actorId: server.id,
+      orLocalVideos: true
+    },
+    nsfw,
+    // Prevent podcast feeds from listing videos in other instances
+    // helps prevent duplicates when they are indexed -- only the author should control them
```

Comment:
Essentially this boils down to the concept of ownership, duplicate feeds, and the user experience in the end user application.

Feeds get user-submitted and often automatically crawled for inclusion into podcast indexes such as https://podcastindex.org.  When multiple feeds of one podcast get submitted there comes into a question of who owns it and which one is "correct".

For the same reasons I don't allow additional URL parameters because all of the different parameters get included as duplicate.

That said, I'm actually not sure how relevant this is on the channel level.  Querying by the video channel ID will only include the local videos anyway, right?